docs/charts.md

```diff
@@ -229,6 +229,38 @@ Managing charts with `requirements.yaml` is a good way to easily keep
 charts updated, and also share requirements information throughout a
 team.
 
+#### Alias field in requirements.yaml
+
+In addition to the other fields above, each requirements entry may contain
+the optional field `alias`.
+
+Adding an alias for a dependency chart would add another copy
```

Comment:
BTW, it's slightly confusing to have the first alias make a copy, rather than just alias (or rename) the first copy. e.g. if you're using alias to pretty up access, but still only need a single instance of the chart, there's no way to do that. So `alias` is acting more like `additional` or some other semantics.

I'll take it, as my primary need was having 2 instances of the same chart, but thought I'd point it out.